### PR TITLE
Changes in hive and lib thirf

### PR DESCRIPTION
**Pull request by balvant-yadav**
1- Hive version changed to 0.13.0
2- Lib thrift version chnaged to 0.8.0
3- Replaced getMapRedWork method with getMapWork in HiveMultiInputFormat

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -81,7 +81,7 @@
     <apache.pig.version>0.11.1</apache.pig.version>
     <apache.crunch.hadoop1.version>0.8.2</apache.crunch.hadoop1.version>
     <apache.crunch.hadoop2.version>0.8.2-hadoop2</apache.crunch.hadoop2.version>
-    <apache.hive.version>0.8.0</apache.hive.version>
+    <apache.hive.version>0.13.0</apache.hive.version>
```
Comment (pavankumar526) [inline pom.xml]: Can't we take inmobi hive version 0.13.3-inm ?

File: hive/src/main/java/com/twitter/elephantbird/mapred/input/HiveMultiInputFormat.java [hunk 2]
```diff
@@ -47,7 +47,7 @@ private void initialize(FileSplit split, JobConf job) throws IOException {
     if (!"".equals(HiveConf.getVar(job, HiveConf.ConfVars.PLAN))) {
       // Running as a Hive query. Use MapredWork for metadata.
       Map<String, PartitionDesc> partitionDescMap =
-          Utilities.getMapRedWork(job).getPathToPartitionInfo();
+          Utilities.getMapWork(job).getPathToPartitionInfo();
```
Comment (pavankumar526) [inline hive/src/main/java/com/twitter/elephantbird/mapred/input/HiveMultiInputFormat.java:50]: Can you please make sure functionality is same

**Comment by pavankumar526** [comment]
Please check tests are passing as well .

File: hive/src/main/java/com/twitter/elephantbird/mapred/input/HiveMultiInputFormat.java (hunk 2 above)
Comment (balvant-yadav) [inline hive/src/main/java/com/twitter/elephantbird/mapred/input/HiveMultiInputFormat.java:50]: Functionality is same.

**Comment by balvant-yadav** [comment]
@pavankumar526  All tests are passing

**Comment by pavankumar526** [comment]
+1 thanks @balvant-yadav

**Pull request merged**